app/topics/topics.module.ts

```diff
@@ -3,36 +3,48 @@ import { ReactiveFormsModule, FormsModule } from '@angular/forms';
 import { BrowserModule } from '@angular/platform-browser';
 
 import { MyTopicsComponent } from './my-topics-list/my-topics-list.component';
-import { NewTopicComponent } from './new-topic/new-topic.component';
-import { ShowTopicComponent } from './show-topic/show-topic.component';
+import { NewTopicComponent } from './topic-management/new-topic/new-topic.component';
 import { TopicTitleComponent } from './shared/topic-title.component';
 import { TopicService } from './shared/topic.service';
 import { topicRouting } from './topics.routing';
 import { MaterialModule } from '../material/material.module';
-import { SharedModule } from '../shared/shared.module';
 import { TextareaComponent } from '../shared/textarea/textarea.component';
+import { TopicInputComponent } from './topic-management/topic-input/topic-input.component';
+import { ShowTopicComponent } from './show-topic/show-topic.component';
+import { TranslateModule } from 'ng2-translate';
+import { TopicListComponent } from './shared/topic-list/topic-list.component';
+import { EditTopicComponent } from './topic-management/edit-topic/edit-topic.component';
+
+import { TagInputComponent } from '../shared/taginput/taginput.component';
+import { TagInputModule } from 'ng2-tag-input';
 import { AllTopicsComponent } from './all-topics/all-topics.component';
 import { TreeView } from './shared/treeview-node/treeview-node.component';
```

Comment:
It was not followed before(Translate etc.). So, I couldn't notice the pattern. Anyways I'll change for TagInut